src/app/features/bracket/bracket.component.ts

```diff
@@ -124,29 +133,31 @@ export class BracketComponent implements OnInit {
             code: team ? team.code : '?',
             score: team && team.score ? team.score : '?'
           });
+        } else {
```

Comment:
No need `else` in this case!

Reply:
Tại sao ko dùng else anh, em thấy nếu có team thì bị push 2 lần.

Reply:
Nếu có team thì code và score = team.code và team.score
Nếu ko có team thì dấu ?
2 dòng `code: team? team.code : '?'` và `score: team && team.score ? team.score '?'` đã bao gồm cho cái else luôn rồi.